docs/admininstrator-guide/permissions.md

```diff
@@ -6,13 +6,14 @@ description: A general note about Jira and Mattermost permissions
 
 ## Can I restrict users from creating or attaching Mattermost messages to Jira issues?
 
-Yes, there is a plugin setting to disable that functionality
+Yes, there is a plugin setting to disable that functionality.
 
 ## How does Mattermost know which issues a user can see?
 
-Mattermost only displays static messages in the channel and does not enforce Jira permissions on viewers in a channel.  Any messages in a channel can be seen by all users of that channel. Subscriptions to jira issues should be made carefully to avoid unwittingly exposing sensitive Jira issues in a public channel for example. Exposure is limited to the information posted to the channel - to transition an issue, or re-assign it - the user would need to have the appropriate permissions in Jira to perform that action.
+Mattermost only displays static messages in the channel and does not enforce Jira permissions on viewers in a channel. 
 
-## Why does each user need to authenticate with Jira?
+Any messages in a channel can be seen by all users of that channel. Subscriptions to Jira issues should be made carefully to avoid unwittingly exposing sensitive Jira issues in a public channel for example. Exposure is limited to the information posted to the channel. To transition an issue or re-assign it the user needs to have the appropriate permissions in Jira.
```

Comment:
```suggestion
Any messages in a channel can be seen by all users of that channel. Subscriptions to Jira issues should be made carefully to avoid unwittingly exposing sensitive Jira issues in a public channel for example. Exposure is limited to the information posted to the channel. To transition or re-assign and issue the user needs to have the appropriate permissions in Jira.
```